internal/pkg/file/file.go

```diff
@@ -42,8 +43,11 @@ func Exists(fileName string) bool {
 // Chown changes file/dir mode
 func Chown(file, owner string, permissions os.FileMode) error {
 	// Chown the file properly for the owner
-	uid, _ := users.GetUID(owner)
-	err := os.Chown(file, uid, -1)
+	uid, err := users.GetUID(owner)
+	if err != nil {
+		logrus.Warnf("failed to get uid for %s, using 0: %v", owner, err)
```

Comment:
Can you use logrus.WithError(err).Warnf(...)?

Reply:
Yep, that's better.

I found over a thousand instances of us logging errors without using WithError (mind the regexes may not be 100% correct), but since logrus is in maintenance mode we probablty don't want to look into this as logrus is in maintenance mode...